cost/azure/rightsize_managed_disks/azure_rightsize_managed_disks.pt

```diff
@@ -1237,6 +1371,15 @@ EOS
         label "IOPS Average %"
         path "iopsPctAverage"
       end
+      field "iopsPctP90" do
+        label "IOPS P90 %"
```

Comment:
These should be formatted similar to other policies.

For example, instead of "IOPS P90 %", it should be "IOPS p90"

See: https://github.com/flexera-public/policy_templates/blob/d4c8705649ebbc65a29ac6027818d16a95e97e19/cost/azure/rightsize_compute_instances/azure_compute_rightsizing.pt#L1632

Reply:
Thank you @XOmniverse , I addressed this change.